id-authentication-default.properties

```diff
@@ -564,19 +560,19 @@ mosip.hotlist.cleanup-schedule.fixed-delay-in-hours=24
 mosip.ida.allowed.enviromemnts=Staging,Developer,Pre-Production,Production
 # Allowed domain Uris. This values should be comma separted.
 #Ex. https://dev.mosip.net,https://qa2.mosip.net
-mosip.ida.allowed.domain.uris=${mosip.api.internal.url}
+mosip.ida.allowed.domain.uris=${mosip.api.internal.url},https://idp.qa-1201-b2.mosip.net
```

Comment:
should be changed to placeholder

Reply:
@ase-101 plz use `https://${mosip.idp.host}` instead of hardcoded domain